### ISSUE TITLE
Custom Mobs Can't Use PT Biped Render

### ISSUE THREAD

**Issue by CrowsOfWar** [CrowsOfWar/PhysicalTraits#6]
You are supposed to be able to apply a PhysicalTraitsModelBiped to any entity, but this won't work!

PhysicalTraitsModelBiped uses player data to retrieve weight, strength, etc. PT player data should support a biped data interface, which requires methods pertaining to the statistics of the player or mob.

Here is what the fix would look like:
```java
/**
 * Some javadocs go here
 * @author CrowsOfWar
 */
public interface PhysicalTraitsBipedData {
  float getWeight();
  void setWeight(float weight);
  // etc
}
```

```java
public class PhysicalTraitsPlayerData extends GoreCorePlayerData implements PhysicalTraitsBipedData {
  // everything else is the same, except add @Override annotations to the relevant methods
}
```

An example of another mod using biped data:
```java
public class MobData implements IEntityExtendedProperties, PhysicalTraitsBipedData {
  // Can now implement methods required for a Physical Traits biped renderer
  // Like this
  @Override
  public float getStrength() {
    return 20; // strong
  }
}
```

**Comment by CrowsOfWar** [comment]
Fix in-progress...

**Comment by CrowsOfWar** [comment]
Fixed by commit 45191057a686af6fdfdc883129531506c7ea8546.
